app/editor/src/features/admin/filters/FilterFormQuery.tsx

```diff
@@ -105,21 +107,52 @@ export const FilterFormQuery: React.FC = () => {
           </p>
         </Row>
         <Row>
-          <FormikText
-            name="settings.search"
-            label="Keywords"
-            value={values.settings.search ?? ''}
-            onChange={(e) => {
-              const value = e.target.value.length ? e.target.value : undefined;
-              updateQuery('search', value);
-            }}
-          >
-            <ToggleGroup
-              className="search-in"
-              options={searchOptions}
-              defaultSelected={values.settings.searchIn ?? searchOptions[0].label}
+          <Col>
+            <FormikTextArea
+              name="settings.search"
+              label="Keywords"
+              value={values.settings.search ?? ''}
+              width={FieldSize.Large}
+              rows={5}
+              onChange={(e) => {
+                const value = e.target.value.length ? e.target.value : undefined;
+                updateQuery('search', value);
+              }}
             />
-          </FormikText>
+          </Col>
+          <Col>
+            <p>
+              The keywords query supports the following operators:
+              <ul>
+                <li>
+                  <code>+</code> signifies AND operation
+                </li>
+                <li>
+                  <code>|</code> signifies OR operation
+                </li>
+                <li>
+                  <code>-</code> negates a single token
+                </li>
+                <li>
+                  <code>"</code> wraps a number of tokens to signify a phrase for searching
+                </li>
+                {/* <li><code>*</code> at the end of a term signifies a prefix query</li> */}
```

Comment:
Probably the case, but just making sure the comments were left intentionally!

Reply:
yes, I left them hidden as I dont think we need to expose those *yet*

Reply:
Does it hurt to expose these now?  This is only visible to admin currently.

Reply:
fair comment.  I am not sure about how the slop and fuzziness works though TBH.